canarytokens/channel_dns.py

```diff
@@ -238,6 +240,18 @@ def query(self, query: Query, src_ip: str):  # noqa C901
         ):
             return defer.succeed(self._do_dynamic_response(name=query.name.name))
 
+        if (canarydrop.type == TokenTypes.CMD) and (
+            invocation_id := src_data["src_data"].get("cmd_invocation_id")
+        ):
+            if any(
+                hit.src_data.get("cmd_invocation_id") == invocation_id
+                for hit in canarydrop.triggered_details.hits
```

Comment:
since this is a inline if it reads easier when on one line
```python
hit.src_data.get("cmd_invocation_id") == invocation_id for hit in canarydrop.triggered_details.hits
```

Reply:
It's auto-formatted by `black` in a pre-commit hook, we can turn it off, but you need two extra lines of comments for that (e.g. in [constants.py](https://github.com/thinkst/canarytokens/blob/master/canarytokens/constants.py#L17)).

Reply:
lets leave it for now, then we can internally chat on it